src/shared/components/banner/index.module.css

```diff
@@ -1,6 +1,10 @@
 @import "shared/styles/imports/variables.css";
 @import "shared/styles/imports/custom-medias.css";
 
+.hidden {
+    display: none;
+}
```

Comment:
It would be better to remove the banner smoothly than removing it with `display: none`.

Reply:
I'll update it later. Not in this PR.